Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt

```diff
@@ -258,6 +258,10 @@ class ConfidenceFeatureProvider private constructor(
         ): ConfidenceFeatureProvider {
             val configuredClient = client ?: ConfidenceRemoteClient(
                 clientSecret = clientSecret,
+                sdkId = "SDK_ID_KOTLIN_PROVIDER",
```

Comment:
Discussions are happening to change the OF SDK naming from `kotlin` to `android`. Shall we follow the same route, we might want to rename this

Reply:
A problem for another PR